### PR TITLE
Configure faf-rabbitmq to be scraped by faf-prometheus

### LINKED ISSUE #132: Configure RabbitMQ for prometheus
https://www.rabbitmq.com/prometheus.html

- [ ] Enable the module `rabbitmq_prometheus` on startup of the rabbitmq container
- [ ] Add rabbitmq to prometheus config

### PR DESCRIPTION
**Pull request by Brutus5000**
fixes #132

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -696,6 +696,8 @@ services:
     image: rabbitmq:3.8.2-management-alpine
     container_name: faf-rabbitmq
     hostname: rabbitmq.${DOMAINNAME}
+    expose:
+      - 15692
```
Comment (norraxx) [inline docker-compose.yml]: We could unify ports as integers, now it's mixed integers and strings.

File: config.template/faf-rabbitmq/enabled_plugins [hunk 2]
```diff
@@ -0,0 +1 @@
+[rabbitmq_management,rabbitmq_prometheus].
```
Comment (norraxx) [inline config.template/faf-rabbitmq/enabled_plugins:1]: Why do we need `rabbitmq_management`?

**Review by norraxx — COMMENTED** [review]
Also I'd like to search for proper dashboard. I didn't test that it, but that one looks good:
https://grafana.com/grafana/dashboards/10991

**Review by norraxx — APPROVED** [review]

File: config.template/faf-rabbitmq/enabled_plugins (hunk 2 above)
Comment (Brutus5000) [reply config.template/faf-rabbitmq/enabled_plugins:1]: rabbitmq_management is the web dashboard. It is currently active and would be overwritten by that change.

File: docker-compose.yml (hunk 1 above)
Comment (Brutus5000) [reply docker-compose.yml]: Done.

**Pull request merged**